src/region/__tests__/RegionAnnotations-test.tsx

```diff
@@ -180,6 +180,28 @@ describe('RegionAnnotations', () => {
             });
         });
 
+        test.each`
+            status                   | shouldRender | isPending | not
+            ${CreatorStatus.init}    | ${false}     | ${false}  | ${'not'}
+            ${CreatorStatus.pending} | ${true}      | ${true}   | ${''}
+            ${CreatorStatus.staged}  | ${true}      | ${false}  | ${''}
+        `(`should $not render reply popup when status is $status`, ({ status, shouldRender, isPending }) => {
+            const wrapper = getWrapper({
+                isCreating: true,
+                staged: getStaged(),
+                status,
+            });
+            wrapper.setState({
+                targetRef: document.createElement('a'),
+            });
+
+            expect(wrapper.exists(PopupReply)).toBe(shouldRender);
+
+            if (shouldRender) {
```

Comment:
These should probably just be separate tests. We also already have a test like this above.